pan-domain-auth-verification/src/test/scala/com/gu/pandomainauth/PanDomainTest.scala

```diff
@@ -16,14 +19,22 @@ class PanDomainTest extends AnyFreeSpec with Matchers with Inside {
   def authStatus(
     cookieData: String,
     validateUser: AuthenticatedUser => Boolean = _ => true,
-    apiGracePeriod: Long = 0,
+    apiGracePeriod: Duration = ZERO,
     system: String = "testsuite",
     cacheValidation: Boolean = false,
     forceExpiry: Boolean = false,
   ) = PanDomain.authStatus(cookieData, OnlyVerification(testPublicKey.key), validateUser, apiGracePeriod, system, cacheValidation, forceExpiry)
 
   "authStatus" - {
-    val authUser = AuthenticatedUser(User("test", "user", "test.user@example.com", None), "testsuite", Set("testsuite"), new Date().getTime + 86400, multiFactor = true)
```

Comment:
Note that [`new Date().getTime`](https://docs.oracle.com/javase/8/docs/api/java/util/Date.html#getTime--) returns a value in _milliseconds_.

`86400` milliseconds is 86.4 seconds, while `86400` seconds is 24 hours, so this strange choice of `86400` probably arose from confusion between seconds and milliseconds - we've chosen not to correct this to a more intentional value here, to demonstrate that the tests still pass with minimal alteration.

Reply:
Now that the tests have been demonstrated to pass (https://github.com/guardian/pan-domain-authentication/actions/runs/13951682077/job/39052570386), can we change it to a more sensible value (presumably `.plusSeconds(86400)`?) by adding an extra commit to the PR?

Reply:
We do have that change added into a subsequent PR (https://github.com/guardian/pan-domain-authentication/pull/185) but I can pull that change onto this one 👍